src/frequenz/sdk/actor/_data_sourcing/_component_metric_request.py

```diff
@@ -46,6 +46,8 @@ def get_channel_name(self) -> str:
             A string denoting a channel name.
         """
         return (
-            f"component-stream::{self.component_id}::{self.metric_id.name}::"
-            f"{self.start_time}::{self.namespace}"
+            f"component_metric_request<namespace={self.namespace},"
+            f"component_id={self.component_id},"
+            f"metric_id={self.metric_id},"
```

Comment:
it would be much nicer to have it say `self.metric_id.name`, because that's the name that we'd use in code.

Reply:
Yeah, just another oversight, sorry :disappointed: